### PR TITLE
Turn code into markdown code blocks for easier readability

### PR DESCRIPTION
**Pull request by ahmadsalim**

### CODE REVIEW

File: Notes/implementation-notes.md [hunk 1]
```diff
@@ -189,9 +190,9 @@ we'll see that the @-pattern name is defined, so turn it into a PLet.
 Names which are bound in types are also bound as @-patterns, meaning that
 functions have access to them. For example, we can say:
 
-vlength : Vect n a -> Nat
-vlength [] = n
-vlength (x :: xs) = n
+    vlength : Vect n a -> Nat
+    vlength [] = n
+    vlength (x :: xs) = n
```
Comment (brendanzab) [inline Notes/implementation-notes.md]: Perhaps you could use fences, like:

````md
```idris
vlength : Vect n a -> Nat
vlength [] = n
vlength (x :: xs) = n
```
````

Comment (ahmadsalim) [reply Notes/implementation-notes.md]: Good idea, thanks!

**Comment by ahmadsalim** [comment]
Sure, good idea!

**Comment by edwinb** [comment]
Thanks!

**Pull request merged**